share/mk/bsd.cpu.mk

```diff
@@ -296,7 +296,7 @@ MACHINE_CPU = sse3
 MACHINE_CPU += amd64 sse2 sse mmx
 ########## arm64
 . elif ${MACHINE_CPUARCH} == "aarch64"
-.  if ${CPUTYPE:Mmorello} != ""
```

Comment:
Hm, `:Mmorello` is a bit weird for CPUTYPE. Above it's `:Mmorello*`, but they should probably both be ==?